### PR TITLE
Whoops

### PR DESCRIPTION
**Pull request by Protopaco**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -58,16 +75,25 @@
...
+      "error",
+      "always"
+    ],
     "max-len": "error",
     "jsx-quotes": "error",
     "babel/no-invalid-this": 1,
     "babel/semi": 0
   }
-}
+}
```
Comment (dbennin125) [inline .eslintrc:99]: new line

File: src/components/app/App.jsx [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+  return (
+    <Router>
+      <Header changePage={changePage} page={page} />
+      <Switch>
+        <Route exact path="/" component={CountriesPage} />
+        <Route path="/country/:code" component={CountryByCode} />
+      </Switch>
+    </Router>
+  );
+}
```
Comment (dbennin125) [inline src/components/app/App.jsx:25]: new line

File: src/components/countries/CountryLIst.test.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom'
+import { render, cleanup } from '@testing-library/react';
+import CountryList from './CountryList';
+
```
Comment (dbennin125) [inline src/components/countries/CountryLIst.test.jsx:5]: alternatively, I saw you had a json file. you could import it and spread it here for a bit more accurate test than just 'f', either is fine.

File: src/components/countries/CountryList.css [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+button:active{
+    filter: drop-shadow(.5px .5px .5px tan)
+}
+
+div ul{
+    display: flex;
+    justify-content: center;
+}
```
Comment (dbennin125) [inline src/components/countries/CountryList.css:34]: new line.

File: src/components/countries/CountryList.jsx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+                    <button
+                        onClick={() => changePage(-1)}>
+                        &lt;
+                    </button>
+                </li>
+                <li data-testid="pageNum">
+                    {page}
+                </li>
+                <li>
+                    <button
```
Comment (dbennin125) [inline src/components/countries/CountryList.jsx:29]: you can add the disabled property to the button like 
```
disabled={page ===1}
```
that way the button is disabled if on the first page.

File: src/components/details/CountryDetail.css [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    color: brown;
+    font-weight: bold;
+}
+
+.frame{
+    width: 100vw;
+    height: 100vh - 200px;
+    display:flex;
+    justify-content: center;
+}
```
Comment (dbennin125) [inline src/components/details/CountryDetail.css:37]: new line.

File: src/components/header/Header.jsx [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+                </button>
+            </Link>
+        </div>
+        <div className={styles.text}>
+            FLAGS OF THE WORLD
+        </div>
+    </div>
+)
+
+export default Header;
```
Comment (dbennin125) [inline src/components/header/Header.jsx:20]: new line.

File: src/components/header/Header.test.jsx [hunk 8]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom'
+import { render, cleanup } from '@testing-library/react';
+import Header from './Header';
+
+describe('Header component', () => {
+    afterEach(() => cleanup());
+    it('renders Header', () => {
+        const { asFragment } = render(
+            <Router>
```
Comment (dbennin125) [inline src/components/header/Header.test.jsx:10]: you can use memoryRouter as well. Either is fine.

File: src/containers/CountriesPage.test.jsx [hunk 9]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```
Comment (dbennin125) [inline src/containers/CountriesPage.test.jsx:1]: this is a great test Paul.

File: src/containers/HeaderPage.jsx [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+import { usePagination } from '../hooks/Countries';
+import Header from '../components/header/Header';
+
+const HeaderPage = () => {
+    const { changePage, page } = usePagination();
+
+    return <Header changePage={changePage} page={page} />
+}
+
+export default HeaderPage;
```
Comment (dbennin125) [inline src/containers/HeaderPage.jsx:11]: new line.

File: src/fixtures/countryByCode.json [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+    "regionalBlocs": [
+        {
+            "acronym": "SAARC",
+            "name": "South Asian Association for Regional Cooperation",
+            "otherAcronyms": [],
+            "otherNames": []
+        }
+    ],
+    "cioc": "AFG"
+}
```
Comment (dbennin125) [inline src/fixtures/countryByCode.json:88]: new line.

File: src/services/countries.js [hunk 12]
```diff
@@ -0,0 +1,22 @@
...
+    return fetch(`${URL}alpha/${code}`)
+        .then(res => res.json())
+        .then(result => ({
+            name: result.name,
+            nativeName: result.nativeName,
+            code: result.code,
+            flagURL: result.flag,
+            capital: result.capital
+        }))
+}
```
Comment (dbennin125) [inline src/services/countries.js:22]: new line.

**Review by dbennin125 — COMMENTED** [review]
Great work Paul!

File: src/components/countries/CountryList.jsx (hunk 5 above)
Comment (Protopaco) [reply src/components/countries/CountryList.jsx:29]: That is a great idea, didn't even occur to me.  Thanks!

**Pull request merged**